tests/base/boring_model.py

```diff
@@ -93,7 +93,7 @@ def step(self, x):
         return out
 
     def training_step(self, batch, batch_idx):
-        output = self.layer(batch)
+        output = self(batch)
```

Comment:
this is necessary to make  boring model work with amp, because the forward gets wrapped